internal/services/cosmos/cosmosdb_account_resource_test.go

```diff
@@ -4166,7 +4221,15 @@ resource "azurerm_cosmosdb_mongo_collection" "test" {
 
   index {
     keys   = ["_id"]
-    unique = true
```

Comment:
Can you share why this was flipped from true to false? It looks like the following is needed but I'm wondering why this needed to be flipped in the first place

```
 // indices can cause test to be inconsistent
  // I believe there is a bug within the azurerm_cosmosdb_mongo_collection that causes inconsistent results on read
  lifecycle {
    ignore_changes = [
      index
    ]
```

Reply:
My mistake for altering the original value. 

In the context of the test, it doesn't make a difference. The test itself fails due to`azurerm_cosmosdb_mongo_collection` flagging changes on the indices during the import phase. This occurs whether set to true or false.

```
        Terraform will perform the following actions:
        
          # azurerm_cosmosdb_mongo_collection.test will be updated in-place
          ~ resource "azurerm_cosmosdb_mongo_collection" "test" {
                id                     = "/subscriptions/...."
                name                   = "acctest-mongodb-coll-240227125500374571"
                # (6 unchanged attributes hidden)
        
              - index {
                  - keys   = [
                      - "_id",
                    ] -> null
                  - unique = true -> null
                }
              + index {
                  + keys   = [
                      + "_id",
                    ]
                  + unique = false
                }
            }
```

I known @rizkybiz  as well as @katbyte both ran into issues on this particular test as well.

Risking this PR becoming stuck due to another resource misbehaving, I opted to ignore the changing property.

Reply:
So weird! But it does look it's not failing because of these changes! Thanks for following up!